Lib/test/test_zipimport.py

```diff
@@ -40,7 +40,7 @@ def make_pyc(co, mtime, size):
         else:
             mtime = int(-0x100000000 + int(mtime))
     pyc = (importlib.util.MAGIC_NUMBER +
-        struct.pack("<iii", 0, int(mtime) & 0xFFFF_FFFF, size & 0xFFFF_FFFF) + data)
+        struct.pack("<iii", 0, int(mtime), size & 0xFFFF_FFFF) + data)
```

Comment:
why this revert?

Reply:
just because @serhiy-storchaka indicated that the signed was used here.

Reply:
I meant that you need to update the format string.